Description: A patch for fit.c before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 Added for raise error-threshold to at least 0.002, because zero
 or negative values lead to algorithm failure.
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Bug-Debian: http://bugs.debian.org/328822
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/fit.c
+++ autotrace-0.31.1/fit.c
@@ -1079,7 +1079,8 @@ fit_with_least_squares (curve_type curve
   spline = best_spline;
   error = best_error;
 
-  if (error < fitting_opts->error_threshold && CURVE_CYCLIC(curve) == false)
+  if ((error < 0.002 || error < fitting_opts->error_threshold)
+      && CURVE_CYCLIC(curve) == false)
     {
       /* The points were fitted with a
          spline.  We end up here whenever a fit is accepted.  We have
